server/app/services/applicant/question/DateQuestion.java

```diff
@@ -47,6 +47,18 @@ public ImmutableList<Path> getAllPaths() {
     return ImmutableList.of(getDatePath());
   }
 
+  @Override
+  public ImmutableMap<Path, String> getJsonEntries() {
+    Path path = getDatePath().asApplicationPath();
+
+    if (getDateValue().isPresent()) {
+      LocalDate date = getDateValue().get();
+      return ImmutableMap.of(path, DateTimeFormatter.ISO_DATE.format(date));
+    } else {
+      return ImmutableMap.of();
+    }
+  }
+
```

Comment:
Above code is taken from `case DATE`:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/export/JsonExporter.java#L121-L219